src/main/java/de/htwberlin/Triplet.java

```diff
@@ -0,0 +1,25 @@
+package de.htwberlin;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Triplet {
+
+    public String Triplet (List<Integer> B,List<Integer> A){
```

Comment:
Variablen per Konvention immer camelCase, also besser sowas wie

```suggestion
    public String Triplet (List<Integer> tripletA, List<Integer> tripletB) {
```